src/main/java/org/jenkinsci/plugins/oic/OicServerWellKnownConfiguration.java

```diff
@@ -219,12 +219,12 @@ public FormValidation doCheckWellKnownOpenIDConfigurationUrl(
         }
 
         @POST
-        public FormValidation doCheckOverrideScopes(@QueryParameter String overrideScopes) {
+        public FormValidation doCheckScopesOverride(@QueryParameter String scopesOverride) {
```

Comment:
@jtnord The form validation for scopes did not work since the jelly uses [`scopesOverride`](https://github.com/jenkinsci/oic-auth-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/oic/OicServerWellKnownConfiguration/config.jelly#L10) as field name. 

Fixed this with this PR or should I open a dedicated PR?